src/main/webapp/assets/js/reviews.js

```diff
@@ -20,22 +20,31 @@ async function getReviews() {
 function createReviewSection(review) {
   const reviewWrapper = document.createElement("div");
   reviewWrapper.className = "review-card";
-
+  
+  let stars = createStars(review.rating);
   reviewWrapper.innerHTML =
     `<div class="review-card-container">
       <div class="name-and-date">
         <p>${review.userID}</p>
-        <div class="rating">
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-        </div>
+        <div class="rating">${stars}</div>
         <p>${review.dateTime}</p>
       </div>
       <p>${review.comment}</p>
     </div>`;
 
   return reviewWrapper;
 }
+
+function createStars(rating) {
+  var i=0;
+  var starHTML = '';
+  for (; i<rating; i++) {
```

Comment:
Nothing wrong but seems like a for loop with if statement may be better?
sth like this:
`  for (let i = 0; i < 5; i++) {
    if (i < rating) {
      starHTML += '<i class="fas fa-star yellow-star"></i>'
    } else {
      starHTML += '<i class="fas fa-star"></i>';
    }
  }`

Reply:
Thanks! That's definitely a lot neater